### ISSUE TITLE
addTable first row has colspan prop cause "addTable: mismatch" issue

### ISSUE THREAD

**Issue by huboo** [gitbrent/PptxGenJS#651]
Version: 3.0.1
Env: Node

Code:
```
slide.addTable(
    [
      [
        {
          text: 'FOOBAR',
          options: { colspan: 2 },
        },
      ],
      [
        {
          text: 'FOO',
          options: { color: '000000' },
        },
        {
          text: 'BAR',
          options: { color: '000000' },
        },
      ],
    ],
    {
      x: 8.97,
      y: 5.347,
      w: 2.76,
      h: 1,
      rowH: 0.292,
      colW: [1.76, 1],
      valign: 'top',
      align: 'center',
      margin: 0,
    },
  );
```

Console message:
```
addTable: mismatch: (colW.length != data.length) Therefore, defaulting to evenly distributed col widths.
```

**Comment by huboo** [comment]
```
// src/gen-objects.ts
// Line: 730
else if (opt.colW && Array.isArray(opt.colW) && opt.colW.length !== firstRowColCnt) {
    console.warn('addTable: mismatch: (colW.length != data.length) Therefore, defaulting to evenly distributed col widths.')
    opt.colW = null
}
```
Like above addTable config I have:
- opt.colW.length = 2
- firstRowColCnt = 1 (because I colspan two col to one)

Then I'll have the warn and column width option reset to null.

**Comment by gitbrent** [comment]
Fixed thanks to @Joshua-rose via Pull #679
